package.json

```diff
@@ -3,7 +3,7 @@
   "version": "4.0.0-beta.1",
   "description": "Keycloak Connect Middleware",
   "homepage": "http://keycloak.org",
-  "main": "index.js",
+  "main": "keycloak.js",
```

Comment:
Not 100% sure this is required to match for the `d.ts` to work, but no harm in doing so.